README.md

```diff
@@ -45,7 +45,7 @@ The following should provide a sense of the operation of the routing scheme:
 
 ```
 pip3 install geopandas numpy pandas xarray netcdf4 
-git clone --progress --single-branch --branch fortran http://github.com/jameshalgren/wrf_hydro_nwm_public.git
+git clone --progress --single-branch --branch master http://github.com/jameshalgren/t-route.git
```

Comment:
this still points to your fork, if that is what you want.  Otherwise it should be https://github.com/noaa-owp/t-route.git

Reply:
right. I wasn't sure what made most sense from the standpoint of committing this. I want my tests to work pointing at my fork. Then I want it to point to owp once the pull is complete.

Reply:
Made the change.